api-spec.yaml

```diff
@@ -0,0 +1,179 @@
+consumes:
+- application/json
+info:
+  description: Column - Ansible as a service
+  title: Column
+  version: 0.0.1
+  contact:
+    website: "https://github.com/vmware/column"
+  license:
+    name: "BSD-2"
+    url: "https://github.com/vmware/column/blob/master/LICENSE"
+produces:
+- application/json
+swagger: "2.0"
+schemes:
+- http
+basePath: /v1
+paths:
+  /runs:
+    post:
+      summary: "Create new run that will execute Ansible playbook"
+      description: ""
+      operationId: "createRun"
+      parameters:
+      - in: "body"
+        name: "body"
+        description: "Run object that will describe execution context"
+        required: true
+        schema:
+          $ref: "#/definitions/RunPayload"
+      responses:
+        202:
+          description: "Run started successfully"
+          schema:
+            $ref: "#/definitions/Run"
+        403:
+          description: "Access forbidden"
+          schema:
+            $ref: "#/definitions/AccessError"
+        default:
+          description: generic error
```

Comment:
Inconsistent. Should either be Generic error or Unknown error or something. Don't use lowercase only on this response.